src/js/core.js

```diff
@@ -1070,8 +1070,11 @@ function MediumEditor(elements, options) {
             return this.options.ownerDocument.execCommand(action, false, null);
         },
 
-        getSelectedParentElement: function () {
-            return Selection.getSelectedParentElement();
+        getSelectedParentElement: function (range) {
+            if (typeof range !== 'undefined') {
+                range = window.getSelection().getRangeAt(0);
```

Comment:
The content window can be passed into medium-editor as an option, so you can use `this.options.contentWindow` instead of `window` here.